.husky/pre-commit

```diff
@@ -0,0 +1 @@
+mvn clean test
```

Comment:
This seems like it will eventually start to run the tests. Is that something that we want?